### PR TITLE
Rename GemPresenter

### PR DESCRIPTION
**Pull request by blowmage**
Use "gem" to indicate that we are building a ruby gem package.

### CODE REVIEW

**Comment by blowmage** [comment]
FWIW, I think this change makes the templates much easier to read. I never really knew what an `api` presenter was, other than a wrapper for the Schema Api object. But since we were hiding that fact from the templates, stuck out to me. I think it makes more sense from the generator's and the templates' perspective to navigate a gem than an api.

**Comment by quartzmo** [comment]
I see your point, especially with removing the `gem_` prefix from all those methods.

File: gapic-generator-cloud/templates/cloud/helpers/presenters/method_presenter.rb [hunk 1]
```diff
@@ -23,13 +23,13 @@ def initialize method
...
   end
 
-  def api
-    service.api
-  end
-
-  def package
-    service.package
-  end
+  # def api
```
Comment (landrito) [inline gapic-generator-cloud/templates/cloud/helpers/presenters/method_presenter.rb]: Why is this chunk of code commented out?

Comment (blowmage) [reply gapic-generator-cloud/templates/cloud/helpers/presenters/method_presenter.rb]: It should have been deleted. It exposes schema objects and we don't want that. I actually thought I did delete it...

Comment (blowmage) [reply gapic-generator-cloud/templates/cloud/helpers/presenters/method_presenter.rb]: Yep, left a commit unpushed. Removed.

**Comment by blowmage** [comment]
Can I get an approval so I can merge?

**Pull request merged**